docs/articles_of_incorporation.md

```diff
@@ -0,0 +1,75 @@
+# Attachment To Articles Of Incorporation Of Catalyst Cooperative
```

Comment:
I think because the [articles of incorporation](https://drive.google.com/drive/folders/1XJZFXtY5QxBGrHwTmgBwlA2pvpClxT2p) is the form and the attachment contains the coop purpose.

Reply:
Ah I see. Huh